packages/react-pose/src/posed.tsx

```diff
@@ -60,7 +60,7 @@ const posed = ((component: React.ComponentType | string) =>
   getComponentFactory(component)) as Posed;
 
 supportedElements.reduce((acc, key) => {
-  acc[key] = getComponentFactory(key);
+  acc[key] = createComponentFactory(key);
```

Comment:
the cache is empty when this reduce takes place, so there is no point to use getComponentFactory which checks the cache